pkg/skaffold/errors/errors.go

```diff
@@ -36,16 +37,16 @@ const (
 	FileSync    = Phase("FileSync")
 	DevInit     = Phase("DevInit")
 	Cleanup     = Phase("Cleanup")
-
-	// Report issue text
-	reportIssueText = "If above error is unexpected, please open an issue https://github.com/GoogleContainerTools/skaffold/issues/new to report this error"
 )
 
 var (
-	setOptionsOnce sync.Once
-	skaffoldOpts   config.SkaffoldOptions
+	// Report issue text
+	reportIssueText = fmt.Sprintf("If above error is unexpected, please open an issue %s to report this error", constants.GithubIssueLink)
+
+	setRunContextOnce sync.Once
+	runCtx            runcontext.RunContext
```

Comment:
Is there any way we can avoid singletons/globals like this?  IMHO if these are needed, they should be pushed into cmd/skaffold/app/cmd and passed in from `ShowAIError()`.

Reply:
Let me see what i can do here.